composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.5",
```

Comment:
It is weird though that travis doesn't fail installing dependencies for the master branch...